lstchain/tools/lstchain_create_dl3_file.py

```diff
@@ -159,16 +160,43 @@ def setup(self):
 
         self.log.debug(f"Output DL3 file: {self.output_file}")
 
+        try:
+            with fits.open(self.input_irf) as hdul:
+                self.use_energy_dependent_cuts = (
+                    "GH_CUT" not in hdul["EFFECTIVE AREA"].header
+                )
+        except KeyError:
```

Comment:
This KeyError won't be raised here now, but probably only later when applying the cuts

Reply:
Is it better to not explicitly mention any Error in this case?

Reply:
Ok. Thanks for clarifying